docs/Cloud_integrations/Device_Integration_Center/webhooks.md

```diff
@@ -60,17 +56,16 @@ Webhook parameters include:
 
 To validate the webhook parameters, it is possible to **test the event handler** before creating the event handler.
```

Comment:
To validate the webhook parameters, you can test the event handler before creating the event handler. (no bold is needed)